doc/source/parameters/dem/model_parameters.rst

```diff
@@ -41,65 +37,116 @@ In this subsection, DEM simulation parameters are defined. These parameters incl
+---------------------------
+Dynamic disabling contacts
+---------------------------
 
-* Lethe-DEM has two ``linear`` and ``non-linear`` particle-wall models.
+The dynamic disabling controls the disabling contact mechanism for performance enhancement. This feature dynamically searches for cells with low particle motion (granular temperature), disabling the computation of contacts for particles within these cells.
 
-* ``euler`` (1st order) and ``velocity-verlet`` (2nd order) are the available integration methods in Lethe-DEM.
+* ``enable dynamic disabling contacts`` enables the feature.
 
-* Three rolling resistance models are available in Lethe-DEM: ``no_resistance``, ``constant_resistance``, ``viscous_resistance``.
+* ``granular temperature threshold`` is the threshold of the granular temperature below which the contacts are disabled.
+* 
+* ``solid fraction threshold`` is the minimum solid fraction of the cell in which the contacts may be disabled.
 
-* ``dynamic disabling contacts`` subsection controls the disabling contact mechanism for performance enhancement. This feature dynamically searches for cells with low particle motion (granular temperature), disabling the computation of contacts for particles within these cells.
-  ``enable dynamic disabling contacts`` enables the feature, ``granular temperature threshold`` is the threshold of the granular temperature below which the contacts are disabled, and ``solid fraction threshold`` is the minimum solid fraction of the cell in which the contacts may be disabled.
-  Some parameters in the load balance section may be used to improve the performance of the dynamic disabling contacts feature using the dynamic load balancing.
-  Note: The ``load balance method`` may be set to ``dynamic_with_disabling_contacts`` and factors of the weight of the cells by mobility status (only active and inactive, mobile factor is always 1) may be adjusted using the ``load balance active weight factor`` and ``load balance inactive weight factor`` parameters.
\ No newline at end of file
+Some parameters in the load balance section may be used to improve the performance of the dynamic disabling contacts feature using the dynamic load balancing.
+Note: The ``load balance method`` may be set to ``dynamic_with_disabling_contacts`` and factors of the weight of the cells by mobility status (only active and inactive, mobile factor is always 1) may be adjusted using the ``load balance active weight factor`` and ``load balance inactive weight factor`` parameters.
```

Comment:
```suggestion
Note: The ``load balance method`` may be set to ``dynamic_with_disabling_contacts`` and factors of the weight of the cells by mobility status may be adjusted using the ``active weight factor`` and ``inactive weight factor`` parameters. There is factor only for active and inactive status, mobile factor is always 1. 
```

Reply:
Also, this could be a note/tip box. I didn't do it before because this wasn't a subsection (note box in a bullet list render is ugly), but now it would be good!